.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,21 @@
 
 ## Checklist
 
-- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
-- [ ] The title of the PR is clear and informative
-- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
+[ ] I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and below information
```

Comment:
```suggestion
- [ ] I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and below information
```

Reply:
I think this line is different from others. It is making a start and all terms are listed below.

Reply:
Yes, but you need to combine `-` and `[ ]` together to render a checklist in markdown

Reply:
Thanks. I have fixed it.